ui/src/app/app.module.ts

```diff
@@ -21,43 +21,42 @@ import {StatusCheckService} from './services/status-check.service';
-import {WorkspaceLibraryComponent} from './views/workspace-library';
-import {WorkspaceListComponent} from './views/workspace-list';
-import {WorkspaceNavBarComponent} from './views/workspace-nav-bar';
-import {WorkspaceShareComponent} from './views/workspace-share';
+import {BreadcrumbComponent} from './components/breadcrumb';
+import {AdminReviewWorkspaceComponent} from './pages/admin/admin-review-workspace';
+import {AdminUserComponent} from './pages/admin/admin-user';
+import {NewNotebookModalComponent} from './pages/analysis/new-notebook-modal';
+import {NotebookListComponent} from './pages/analysis/notebook-list';
+import {NotebookRedirectComponent} from './pages/analysis/notebook-redirect/component';
+import {ResetClusterButtonComponent} from './pages/analysis/reset-cluster-button';
+import {AppComponent, overriddenUrlKey} from './pages/app/component';
+import {CohortActionsComponent} from './pages/data/cohort/cohort-actions';
+import {CohortListComponent} from './pages/data/cohort/cohort-list';
+import {ConceptAddModalComponent} from './pages/data/concept/concept-add-modal';
+import {ConceptHomepageComponent} from './pages/data/concept/concept-homepage';
+import {ConceptSetActionsComponent} from './pages/data/concept/concept-set-actions';
+import {ConceptSetDetailsComponent} from './pages/data/concept/concept-set-details';
+import {ConceptSetListComponent} from './pages/data/concept/concept-set-list';
+import {ConceptTableComponent} from './pages/data/concept/concept-table';
+import {ErrorHandlerComponent} from './pages/error-handler/component';
+import {HomepageComponent} from './pages/homepage/homepage';
+import {QuickTourModalComponent} from './pages/homepage/quick-tour-modal';
+import {RecentWorkComponent} from './pages/homepage/recent-work';
+import {InitialErrorComponent} from './pages/initial-error/component';
+import {PageTemplateSignedOutComponent} from './pages/login/page-template-signed-out/component';
+import {SignInComponent} from './pages/login/sign-in';
+import {DataUseAgreementComponent} from './pages/profile/data-use-agreement';
+import {ProfilePageComponent} from './pages/profile/profile-page';
+import {RoutingSpinnerComponent} from './pages/routing-spinner/component';
+import {SignedInComponent} from './pages/signed-in/component';
+import {TopBoxComponent} from './pages/top-box/component';
+import {StigmatizationPageComponent} from './pages/workspace/stigmatization-page';
+import {WorkspaceAboutComponent} from './pages/workspace/workspace-about';
+import {WorkspaceEditComponent} from './pages/workspace/workspace-edit';
+import {WorkspaceLibraryComponent} from './pages/workspace/workspace-library';
+import {WorkspaceListComponent} from './pages/workspace/workspace-list';
+import {WorkspaceNavBarComponent} from './pages/workspace/workspace-nav-bar';
+import {WorkspaceShareComponent} from './pages/workspace/workspace-share';
+import {BugReportComponent} from './widgets/bug-report';
```

Comment:
"widgets/" with this single component feels like a bit of a cop-out... There are other fairly specific modals living in `components/` - any reason not to put it there too?

Reply:
Sure, will do.